source/_data/plugins.yml

```diff
@@ -2419,3 +2419,11 @@
     - delopyer
     - qiniu
     - 七牛云
+- name: hexo-renderer-scss
+  description: "A hexo plugin for node-scss"
+  link: https://github.com/o2team/hexo-renderer-scss
```

Comment:
It redirects to https://github.com/mamboer/hexo-renderer-scss.

Reply:
Correct